SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslationManager.cpp

```diff
@@ -59,7 +59,8 @@ void SpatialVirtualWorkerTranslationManager::WriteMappingToSchema(Schema_Object*
 	{
 		Schema_Object* EntryObject = Schema_AddObject(Object, SpatialConstants::VIRTUAL_WORKER_TRANSLATION_MAPPING_ID);
 		Schema_AddUint32(EntryObject, SpatialConstants::MAPPING_VIRTUAL_WORKER_ID, Entry.Key);
-		SpatialGDK::AddStringToSchema(EntryObject, SpatialConstants::MAPPING_PHYSICAL_WORKER_NAME, Entry.Value);
+		SpatialGDK::AddStringToSchema(EntryObject, SpatialConstants::MAPPING_PHYSICAL_WORKER_NAME, Entry.Value.Get<0>());
+		Schema_AddEntityId(EntryObject, SpatialConstants::MAPPING_SERVER_WORKER_ENTITY_ID, Entry.Value.Get<1>());
```

Comment:
Why was this added?

Reply:
Because we need to know, based on the strategized virtual worker ID for the PlayerStart, where to send the ForwardSpawnRequest command

Reply:
Key/Value here as well for consistency?

Reply:
Yeah I prefer key/value in general